### ISSUE TITLE
java.lang.IllegalStateException: Timer already cancelled

### ISSUE THREAD

**Issue by johny65** [orientechnologies/orientdb#9497]
## Expected behavior  

Disconnect from database without errors.

## Actual behavior  

When connected to a database in plocal mode through the console, almost every time when I disconnect from it, it throws `java.lang.IllegalStateException: Timer already cancelled.`

## Steps to reproduce  

```
$ orientdb> connect plocal:/path/to/database admin admin;
Connecting to database [plocal:/path/to/database] with user 'admin'...OK
$ orientdb {db=database}> (do some stuff...)
$ orientdb {db=database}> disconnect;
Disconnecting from the database [database]...OK
java.lang.IllegalStateException: Timer already cancelled.
	at java.base/java.util.Timer.sched(Timer.java:398)
	at java.base/java.util.Timer.schedule(Timer.java:194)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.scheduleOnce(OrientDBEmbedded.java:1074)
	at com.orientechnologies.orient.core.db.viewmanager.ViewManager.schedule(ViewManager.java:151)
	at com.orientechnologies.orient.core.db.viewmanager.ViewManager.updateViews(ViewManager.java:163)
	at com.orientechnologies.orient.core.db.viewmanager.ViewManager.access$400(ViewManager.java:55)
	at com.orientechnologies.orient.core.db.viewmanager.ViewManager$1.lambda$run$0(ViewManager.java:146)
	at com.orientechnologies.orient.core.db.OrientDBEmbedded.lambda$executeNoAuthorization$9(OrientDBEmbedded.java:1092)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by tglman** [comment]
Hi,

This should be already fixed in 3.2.x series.

Closing

**Comment by a-drop-of-tears** [comment]
![image](https://github.com/user-attachments/assets/72113efe-00a4-4136-990b-83e50481d888)
Hi,
I encountered this error in version 3.2.29 @tglman
